src/constants/networks.ts

```diff
@@ -1,12 +1,22 @@
+import type { CaipChainId } from '@metamask/utils';
+
 export const FEATURED_NETWORKS = {
-  'eip155:1': 'Ethereum Mainnet',
-  'eip155:59144': 'Linea Mainnet',
-  'eip155:42161': 'Arbitrum One',
-  'eip155:43114': 'Avalanche Network C-Chain',
-  'eip155:56': 'BNB Chain',
-  'eip155:10': 'OP Mainnet',
-  'eip155:137': 'Polygon Mainnet',
-  'eip155:324': 'zkSync Era Mainnet',
-  'eip155:8453': 'Base Mainnet',
-  'eip155:1337': 'Localhost',
+  'Ethereum Mainnet': 'eip155:1',
+  'Linea Mainnet': 'eip155:59144',
+  'Arbitrum One': 'eip155:42161',
+  'Avalanche Network C-Chain': 'eip155:43114',
+  'BNB Chain': 'eip155:56',
+  'OP Mainnet': 'eip155:10',
+  'Polygon Mainnet': 'eip155:137',
+  'zkSync Era Mainnet': 'eip155:324',
+  'Base Mainnet': 'eip155:8453',
+  Localhost: 'eip155:1337',
+  'Solana Mainnet': 'solana:5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp',
 } as const;
+
+export const getNetworkName = (chainId: CaipChainId): string => {
+  const entry = Object.entries(FEATURED_NETWORKS).find(
+    ([_, id]) => id === chainId,
+  );
```

Comment:
[nit]: since we have no need for the keys here, I would suggest:
```suggestion
  const entry = Object.values(FEATURED_NETWORKS).find(
    (id) => id === chainId,
  );
```

Reply:
I committed this but the reason we want it to be entries is because we want the full entry to be returned by the `.find()`